### ISSUE TITLE
代码错误

### ISSUE THREAD

**Issue by GuilmK** [WenmuZhou/PSENet.pytorch#63]
感谢作者提供的优秀代码，在训练过程中我遇到一旦出现Loss小于0.4，运行一次测试集上的测试后，代码就会阻塞，无法继续训练。在对问题进行追踪后，我发现是DataLoader的写法有问题，导致在ubuntu上的多线程阻塞。解决方法：
1. config.py 将workers数量改为0，该方法会让数据读取变慢
2. train.py中的train_epoch里
    for i, (images, labels, training_mask) in enumerate(train_loader):
改成
    for i, mydatas in enumerate(train_loader):
               (images, labels, training_mask) = mydatas
即可解决

**Comment by WenmuZhou** [comment]
可以提个pr

**Comment by 1156150793** [comment]
为什么我用作者提供的weight。   AP为0，精确度也好低

**Comment by interestingzhuo** [comment]
为什么第二种方法还是会卡住？
